include/crm/common/results.h

```diff
@@ -108,6 +108,7 @@ enum pcmk_rc_e {
     /* When adding new values, use consecutively lower numbers, update the array
      * in lib/common/results.c, and test with crm_error.
      */
+    pcmk_rc_bz2_software        = -1040,
```

Comment:
I'd be more generic e.g. rc_compression

Reply:
Renamed and added a better string representation.